LiveLines.Api/Lines/Privacy.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace LiveLines.Api.Lines;
+
+public enum Privacy
```

Comment:
Privacy is quite generic, since this relates to the privacy of lines, perhaps `LinePrivacy`?